### PR TITLE
Implements the simple linked list exercise in PHP

### PR DESCRIPTION
**Pull request by camilopayan**

### CODE REVIEW

**Comment by kytrinyx** [comment]
@petemcfarlane Will you have a moment to review this before the end of October (tomorrow)? I want to make sure that @camilopayan qualifies for the Hacktoberfest t-shirt.

**Comment by kytrinyx** [comment]
@camilopayan if you put this not-quite-last in the config.json it might stop having conflicts, even if we merge other PRs.

**Comment by camilopayan** [comment]
That worked! I think only PRs matter for Hacktoberfest, so I'd qualify either way. Thanks for the help, though 😀

**Comment by camilopayan** [comment]
Ah whoops, I think I conflated the exercises for `linked-list` and `simple-linked-list`, which just requires implementing a LIFO stack as a linked list. I'll change the labels to `simple-linked-list`

**Comment by camilopayan** [comment]
let me know if that clears that up @petemcfarlane

File: exercises/simple-linked-list/simple-linked-list_test.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    public function testCanReturnListAsArray()
+    {
+        $this->markTestSkipped();
+        $list = new LinkedList();
+        $list->push(8);
+        $list->push(5);
+        $list->push(1);
+        $this->assertEquals([1, 5, 8], $list->asArray());
```
Comment (petemcfarlane) [inline exercises/simple-linked-list/simple-linked-list_test.php]: It looks good, but I'm still pretty sure the order should be `[8, 5, 1]`, not `[1, 5, 8]`. Maybe you could use the reverse function inside `asArray` to do the work for you?

Comment (camilopayan) [inline exercises/simple-linked-list/simple-linked-list_test.php]: The [problem description ](https://github.com/exercism/x-common/blob/master/exercises/simple-linked-list/description.md) describes a Last In, First Out stack, so 1, being the last in, would head the array, then 5, then 8. I think the order is correct for that.

otoh there are parts of the exercise that are missing, like the reversal function and creating the list from an array, so I'll get to that. Probably shouldn't have copied off the java tests for this 😅

**Comment by kytrinyx** [comment]
Since it's been over a year on this one, I'm going to go ahead and close it.

@camilopayan we're _so_ close, so if you'd like to branch off of a recent master and submit this again taking @petemcfarlane's comments into account, we'd be delighted to take another stab at getting this merged!

**Pull request closed without merging**